### PR TITLE
Add why stoat page in the doc

### LINKED ISSUE #52: Add a "why stoat" section in the doc

### PR DESCRIPTION
**Pull request by tuliren**
Resolve #52.

### CODE REVIEW

File: docs/docs/company/team.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ As the first engineer, he helped to build and grow the open source project from
 
 ### [Liren Tu](https://www.linkedin.com/in/tuliren/)
 
-Liren is a top-notch engineering manager, architect, and builder. 
+Liren is a software engineer.
```
Comment (jrhizor) [inline docs/docs/company/team.md:18]: Selling yourself short Liren. I had ChatGPT jazz it up and got each of our intro lines. I really think you should keep it.

File: docs/docs/intro.md [hunk 2]
```diff
@@ -38,22 +38,14 @@ data such as test runtimes or any other values that are computed as part of the
...
-## Templating
-
-Want to summarize any data from your builds? Try out our [templating tutorial](tutorials/templating)!
-
-## CLI
+| Plugin | Description | Doc |
+| --- | --- | --- |
+| Static hosting | Host any build artifacts for preview. | [link](tutorials/static-hosting) |
+| Job runtime | Track the runtime of each job in a workflow. | [link](tutorials/job-runtime) |
+| Template | Render a custom template using data from the build. | [link](tutorials/templating) |
```
Comment (jrhizor) [inline docs/docs/intro.md:49]: This isn't actually a plugin. Maybe just say this is a table of the different tutorials we have?

**Review by jrhizor — APPROVED** [review]

File: docs/docs/company/team.md (hunk 1 above)
Comment (tuliren) [reply docs/docs/company/team.md:18]: Alright. I won't remove all of it. Some of the description actually was incorrect. For example, I did not reduce the time to create API integrations by hours. That was Sherif and Alex G, who authored the no / low-code CDK.

Comment (jrhizor) [reply docs/docs/company/team.md]: Oh I was basing that on "Introduced code template for data connector creation. Shortened the time to write a new connector from hours to seconds." from LinkedIn.

**Pull request merged**